lesson08/assignment/vue-cli/src/App.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+<div id="app">
+ <div id="nav">
```

Comment:
HTML5 semantic elements are the solution to needing an easily selectable nav element. Use an `<nav>` element here, rather than a div with id of "nav".